test_fms/diag_manager/test_diag_manager2.sh

```diff
@@ -804,6 +804,420 @@ _EOF
+    corner3:
+    corner4:
+  global_meta:
+  - {}
+- file_name: file5
+  freq: 6 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  freq_units: hours null null null null null null null null null null null
+  time_units: hours
+  unlimdim: time
+  new_file_freq: -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  new_file_freq_units: null null null null null null null null null null null null
+  start_time:
+  file_duration: -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  file_duration_units: null null null null null null null null null null null null
+  varlist:
+  - module: atm_mod
+    var_name: var4
+    reduction: average
+    kind: r4
+    output_name:
+    long_name:
+    units:
+    zbounds: -999.00, -999.00
+    n_diurnal: 0
+    pow_value: 0
+    dimensions: time z y3_sub01 x3_sub01
+  sub_region:
+  - grid_type: index
+    tile: 1
+    corner1: 10 15
+    corner2: 20 15
+    corner3: 10 25
+    corner4: 20 25
+  global_meta:
+  - {}
+- file_name: file6%4yr%2mo%2dy%2hr
+  freq: 6 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  freq_units: hours null null null null null null null null null null null
+  time_units: hours
+  unlimdim: time
+  new_file_freq: 6 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  new_file_freq_units: hours null null null null null null null null null null null
+  start_time: 2 1 1 0 0 0
+  file_duration: 12 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  file_duration_units: hours null null null null null null null null null null null
+  varlist:
+  - module: ocn_mod
+    var_name: var1
+    reduction: average
+    kind: r4
+    output_name:
+    long_name:
+    units:
+    zbounds: -999.00, -999.00
+    n_diurnal: 0
+    pow_value: 0
+    dimensions: time y x
+  sub_region:
+  - grid_type: null
+    tile: -999
+    corner1:
+    corner2:
+    corner3:
+    corner4:
+  global_meta:
+  - {}
+- file_name: file7
+  freq: 6 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  freq_units: hours null null null null null null null null null null null
+  time_units: hours
+  unlimdim: time
+  new_file_freq: -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  new_file_freq_units: null null null null null null null null null null null null
+  start_time:
+  file_duration: -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  file_duration_units: null null null null null null null null null null null null
+  varlist:
+  - module: ocn_mod
+    var_name: var1
+    reduction: none
+    kind: r4
+    output_name:
+    long_name:
+    units:
+    zbounds: -999.00, -999.00
+    n_diurnal: 0
+    pow_value: 0
+    dimensions: time y x
+  sub_region:
+  - grid_type: null
+    tile: -999
+    corner1:
+    corner2:
+    corner3:
+    corner4:
+  global_meta:
+  - {}
+- file_name: file8%4yr%2mo%2dy%2hr%2min
+  freq: 1 1 1 -999 -999 -999 -999 -999 -999 -999 -999 -999
+  freq_units: hours hours hours null null null null null null null null null
```

Comment:
Why all the nulls and -999?

Reply:
They are just printing what they got initialized to (DIAG_NULL=-999), and then i had the units just write out null. Do you think we should just leave out any defaults? I had debugging in mind but there's probably better ways to do that anyway.

Reply:
yeah I think i we should leave out any defaults

Reply:
@uramirez8707 Should be fixed up now